terminus.yaml

```diff
@@ -0,0 +1,81 @@
+## TERMINUS
```

Comment:
Looks like this file was checked in accidentally -- mind removing it?

Reply:
WHOOPS, thanks.